### DISCUSSION TITLE
Search all of github (on code) pagination not working

### DISCUSSION THREAD

**Discussion by kabundege** [community/community#138366] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

# Description?
Pagination navigation is not working on all github searh

# Steps to reproduction?
1. all github search on this query `axiosInstance.interceptors.response.use`
2. click the code section, if its not initially selected (not the repository, issues, ..etc)
3. scroll down to the pagination, and click on page 2
4. i get redirected to the repository section, when i expected to go to the second page of the code section

# Visual AID 

https://github.com/user-attachments/assets/92f16e67-3d8c-41f3-a1f2-35960270a6a4

**Comment by ferid333** [comment]
I just checked and there is no problem for me. You can try again or just refresh the page
